pkg/job-queue/Doctrine/JobStorage.php

```diff
@@ -39,9 +39,13 @@ class JobStorage
     private $uniqueTableName;
 
     /**
-     * @param ManagerRegistry $doctrine
-     * @param string          $entityClass
-     * @param string          $uniqueTableName
+     * @var string
+     */
+    private $entityManagerName;
```

Comment:
I think it's some leftover code from when there was still the concept of setting a particular entity manager?